Godeps/_workspace/src/k8s.io/kubernetes/cmd/kubelet/app/server.go

```diff
@@ -681,6 +687,7 @@ func SimpleKubelet(client *client.Client,
 		ResolverConfig:      kubelet.ResolvConfDefault,
 		ResourceContainer:   "/kubelet",
 		RootDirectory:       rootDir,
+		SerializeImagePulls: true,
```

Comment:
@eparis That's only for the 'simple' method, which AFAIK is only used for testing now.  The default values for the CLI flag is going to be 'false', which is the opposite of what this PR says it does.